polygonwm.c

```diff
@@ -328,18 +332,94 @@ spiral(Monitor *mon) {
 	fibonacci(mon, 0);
 }
 
+void dewdrops(Monitor *mon) {
+	unsigned int i, n, nx, ny, nw, nh;
+	Client *c;
+
+	for(n = 0, c = nexttiled(mon->clients); c; c = nexttiled(c->next), n++);
+	if(n == 0) {
+		return;
+	}
+	nx = mon->wx;
```

Comment:
Need comments here. No idea what nx and ny mean. Suckless is known for using bad variable name and not commenting enough. Let us not do the same

Reply:
Okay. Will do.